pxr/imaging/hdSt/testenv/testHdStMaterialXShaderGen/baseline/shadergen_UsdPSdefault.out

```diff
@@ -70,7 +70,6 @@ float displacement;
 float occlusion;
 
 // Uniform block: PrivateUniforms
-float u_alphaThreshold = 0.001000;
```

Comment:
These were incorrectly detected as transparent by MaterialX. The new code that enforces transparency using the materialTag is correct so I had to fix the baseline.